### PR TITLE
PR for g.region.area 

### PR DESCRIPTION
**Pull request by Momen-Mawad**
Includes the reviewed code of g.region.area from Gitlab

### CODE REVIEW

**Review by mmacata — APPROVED** [review]
Great, thanks!

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# E501 line too long (93 > 88 characters)
+# F821 undefined name '_'
+
+exclude = .git
+
+max-line-length = 88
+
+per-file-ignores =
+    ./g.region.area.py: F821, E501
+    ./testsuite/test_g_region_area.py: F821
```
Comment (neteler) [inline .flake8:12]:
```suggestion
    ./testsuite/test_g_region_area.py: F821
```

(missing newline added)

**Comment by Momen-Mawad** [comment]
Added new line to .flake8

**Review by neteler — APPROVED** [review]
Thanks!

**Pull request merged**